src/features/inbox/events/message_response.ts

```diff
@@ -9,18 +10,19 @@ import {
   messageDoneAction,
 } from "../data";
 
-slack.event(messageDoneAction, async (request) => {
-  const payload: { actions: [{ value: string }] } = request.payload;
-  const action = payload.actions[0]; // assuming the button is the first action
-  const actionData = JSON.parse(action.value);
+slack.action(
+  messageDoneAction.action_id || messageDismissedAction.action_id,
```

Comment:
Does this work? It reads like its just always using 'doneAction'. Probably needs a separate action handler for each action.